### DISCUSSION TITLE
add lesson

### DISCUSSION THREAD

**Discussion by teguhpermadi** [sanidhyy/duolingo-clone#18] [Q&A] [answered]
is it possible to create custom lesson in this app, for example math lessons

**Comment by sanidhyy** [comment, marked as answer]
**Yes, this app includes a fully functional admin dashboard where you can easily create, add, edit, or delete courses, lessons, and questions.**

To access the admin dashboard, follow these steps:
1. After setting up the app, go to the Clerk dashboard and retrieve the admin **User ID**.
2. Add this **User ID** to your `.env` file as per the instructions in the README.
3. You can then access the admin dashboard via the `/admin` route.

**Reply by teguhpermadi** [reply]
ok thanks I will try it
